CONTRIBUTING.md

```diff
@@ -0,0 +1,58 @@
+# Requirements
```

Comment:
Contribution file needs a mention of CLA - https://docs.opensource.microsoft.com/content/releasing/accepting-contributions.html follow the instructions in the link

Reply:
I have included this part in Readme as mentioned
```
This can go in the README.md file or in a separate CONTRIBUTING.md file. Below is a sample of the minimum information you should include. Note that the Code of Conduct information is required to be prominent in all Microsoft repositories.
```

and have created project-specific readme